stress-generator/values.yaml

```diff
@@ -0,0 +1,19 @@
+image:
+  repository: kupolua/stress-generator
+  tag: latest
+  env:
+    hostname: HOST
+    hostvalue: stress-demo-api.default.svc.cluster.local
```

Comment:
"...and split deployment variable and api service variable,..."

env:
    host:
      name: HOST
      deployment: stress-demo
      apiservice: api